### ISSUE TITLE
Problem uploading metatainer

### ISSUE THREAD

**Issue by Timmimim** [o2r-project/o2r-meta#44]
I run the o2r microservices locally and try to upload the test ERC `/.../ownCloud/o2r-data/Korpus/Bagtainer/metatainer.zip`. I get the following error in the log of o2r-loader:

```
  loader [K0yfk] Extracting metadata +89ms
  loader [K0yfk] Running metadata extraction with command "python3 ../o2r-meta/o2rmeta.py -debug extract --inputdir /tmp/o2r/compendium/K0yfk/data --outputdir /tmp/o2r/compendium/K0yfk/data/.erc --metafiles --ercid K0yfk --basedir /tmp/o2r/compendium/" +0ms
  loader [K0yfk] Problem during metadata extraction:
  loader 	Command failed: python3 ../o2r-meta/o2rmeta.py -debug extract --inputdir /tmp/o2r/compendium/K0yfk/data --outputdir /tmp/o2r/compendium/K0yfk/data/.erc --metafiles --ercid K0yfk --basedir /tmp/o2r/compendium/
Traceback (most recent call last):
  File "../o2r-meta/o2rmeta.py", line 92, in <module>
    metaextract.start(i=argsd['inputdir'], o=argsd['outputdir'], s=argsd['outputtostdout'], xo=argsd['stayoffline'], e=argsd['ercid'], b=argsd['basedir'], m=argsd['metafiles'], xml=argsd['modexml'])
  File "/home/timmimim/Documents/o2r/o2r-meta/extract/metaextract.py", line 795, in start
    MASTER_MD_DICT['viewfile'].append(get_rel_path(full_file_path))
AttributeError: 'NoneType' object has no attribute 'append'

  loader 	undefined +2s
  loader Error: Command failed: python3 ../o2r-meta/o2rmeta.py -debug extract --inputdir /tmp/o2r/compendium/K0yfk/data --outputdir /tmp/o2r/compendium/K0yfk/data/.erc --metafiles --ercid K0yfk --basedir /tmp/o2r/compendium/
  loader Traceback (most recent call last):
  loader   File "../o2r-meta/o2rmeta.py", line 92, in <module>
  loader     metaextract.start(i=argsd['inputdir'], o=argsd['outputdir'], s=argsd['outputtostdout'], xo=argsd['stayoffline'], e=argsd['ercid'], b=argsd['basedir'], m=argsd['metafiles'], xml=argsd['modexml'])
  loader   File "/home/timmimim/Documents/o2r/o2r-meta/extract/metaextract.py", line 795, in start
  loader     MASTER_MD_DICT['viewfile'].append(get_rel_path(full_file_path))
  loader AttributeError: 'NoneType' object has no attribute 'append'
  loader 
  loader     at ChildProcess.exithandler (child_process.js:198:12)
  loader     at emitTwo (events.js:106:13)
  loader     at ChildProcess.emit (events.js:191:7)
  loader     at maybeClose (internal/child_process.js:891:16)
  loader     at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5) Traceback (most recent call last):
  File "../o2r-meta/o2rmeta.py", line 92, in <module>
    metaextract.start(i=argsd['inputdir'], o=argsd['outputdir'], s=argsd['outputtostdout'], xo=argsd['stayoffline'], e=argsd['ercid'], b=argsd['basedir'], m=argsd['metafiles'], xml=argsd['modexml'])
  File "/home/timmimim/Documents/o2r/o2r-meta/extract/metaextract.py", line 795, in start
    MASTER_MD_DICT['viewfile'].append(get_rel_path(full_file_path))
AttributeError: 'NoneType' object has no attribute 'append'
 [o2rmeta][debug] received arguments: {'basedir': '/tmp/o2r/compendium/', 'ercid': 'K0yfk', 'modexml': False, 'stayoffline': False, 'outputdir': '/tmp/o2r/compendium/K0yfk/data/.erc', 'outputtostdout': False, 'tool': 'extract', 'inputdir': '/tmp/o2r/compendium/K0yfk/data', 'debug': True, 'metafiles': True}
[o2rmeta] launching extractor
[o2rmeta][extract] directory </tmp/o2r/compendium/K0yfk/data/.erc> will be created during extraction...
[o2rmeta][extract] processing /tmp/o2r/compendium/K0yfk/data/Dockerfile
[o2rmeta][extract] processing /tmp/o2r/compendium/K0yfk/data/document.Rmd
[o2rmeta][extract] requesting orcid for <Ted Tester>
[o2rmeta][extract] 200 OK
[o2rmeta][extract] requesting orcid for <Carl Connauthora>
[o2rmeta][extract] 200 OK
[o2rmeta][extract] 3521 bytes written to ../../../../../tmp/o2r/compendium/K0yfk/data/.erc/meta_2017082515225927_document_.json
[o2rmeta][extract] processing /tmp/o2r/compendium/K0yfk/data/bagtainer.yml
[o2rmeta][extract] processing /tmp/o2r/compendium/K0yfk/data/document.html
```

The UI only shows the following error:

```json
{"error":"metadata extraction failed: 'NoneType' object has no attribute 'append'"}
```

o2r-meta version: 

```
o2r-meta$ git rev-parse --verify HEAD
cb0176ed3b0a06d3ed1956ed1ff8f8f2c37a4592

/o2r-meta$ git status
On branch master
Your branch is up-to-date with 'origin/master'.
```

Note that it does work with the version before the last commit:

```
/o2r-meta$ git status
HEAD detached at de9c512
```

**Comment by 7048730** [comment]
looks like the _viewfile_ element should have been initialised as list. I think it is supposed to be _viewfiles_ anyway (which is a list). With all this reverting recently, it seems I mixed up some of the structure of the md dummy.
I try to fix asap...

**Comment by 7048730** [comment]
included with 8154420d8f5d8bdaa2b30f8abadbe6c34742042c
